### ISSUE TITLE
Don't suggest space in a variable name

### ISSUE THREAD

**Issue by THausherr** [codespell-project/codespell#1399]
As shown here
https://fossies.org/linux/test/pdfbox-trunk.zip/codespell.html
"upto" is suggested to be "up to". However in the example, "upto" is a variable.

**Comment by peternewman** [comment]
See also the discussion here:
https://github.com/codespell-project/codespell/issues/1224

And to a lesser extent:
https://github.com/codespell-project/codespell/issues/314

I can't see that specific example on fossies now, but using the ignore list would be the best way to work around it normally (I don't know if you can do it via their system or not).

**Comment by THausherr** [comment]
Yes, he has already put it in the ignore list, discussion [here](https://mail-archives.apache.org/mod_mbox/tika-dev/201912.mbox/%3Calpine.LSU.2.21.1912262103330.1880%40fossies.org%3E).
